### ISSUE TITLE
INSTALL.md still lists qt4 packages

### ISSUE THREAD

**Issue by BenWiederhake** [falkTX/Cadence#241]
tl;dr: Please figure out the correct dependencies and put them into `INSTALL.md`

This is significant because the user (me) has to guess which packages are necessary to run cadence.

It seems that installing the following packages is sufficient (but maybe not necessary):
`pyqt5-dev pyqt5-dev-tools python-pyqt5 qtbase5-dev qdbus-qt5 python3-dbus.mainloop.pyqt5`

Also to note: On Debian, you may run into the following problem:

    $ python3 -c 'from PyQt5 import QtCore'
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
    ImportError: cannot import name 'QtCore'

This is because the package `python3-pyqt5` only provides the file for Python 3.7 (`/usr/lib/python3/dist-packages/PyQt5/QtCore.cpython-37m-x86_64-linux-gnu.so`), so to fix this error upgrade to python 3.7.  I'm sure it can be injected properly into the build, but I didn't look into it.  When running, `python3.7 src/cadence.py` works as intended when the system default is set to python 3.6

**Comment by falkTX** [comment]
This was handled some time ago
